.gitignore

```diff
@@ -160,3 +160,5 @@ haystack/json-schemas
 
 # ruff
 .ruff_cache
+
+.history
```

Comment:
Where does this `.history` comes from? I would rather keep it out unless is something python related.

Reply:
I do not know how to keep it out. Can you do that?

Reply:
ok, I can remove it after it's merged